### PR TITLE
Clean up the ROS example test.

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

**Comment by elopio** [comment]
retest this please

**Comment by elopio** [comment]
This is related to https://bugs.launchpad.net/snapcraft/+bug/1544790
It makes the examples tests pass in the cloud. However in order to reenable them in autopkgtests we first need a proxy exception. I'm requesting it...

File: examples_tests/test_ros.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2015, 2016 Canonical Ltd
```
Comment (sergiusens) [inline examples_tests/test_ros.py]: this should be just 2016

Comment (elopio) [inline examples_tests/test_ros.py]: This is code from 2015, just moved to a separate file now. So I think it should have the two years.

Comment (elopio) [inline examples_tests/test_ros.py]: This is code from 2015, just moved to a separate file now. So I think it should have the two years.

Comment (elopio) [inline examples_tests/test_ros.py]: This is code from 2015, just moved to a separate file now. So I think it should have the two years.

**Comment by kyrofa** [comment]
Closed in favor of #515

**Pull request closed without merging**